Makefile

```diff
@@ -31,7 +31,7 @@ install:
 	echo "Installing is not necessary anymore. It will be done in post-commit."
 
 clean:
-	-rm pep-0000.txt
+	-rm pep-0000.rst
```

Comment:
Old checkouts may still have the text file around, so best to remove both files here for now.

Reply:
`rm non-existent-file` would result in an exit code of 1, which would make `make` abort the build. Adding a `-` before the command would make `make` ignore these errors:

```
clean:
        -rm -f *.o
```

(example from https://www.gnu.org/software/make/manual/html_node/Errors.html)